src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -253,7 +248,8 @@ object DepositHandler {
 
             if (missingFilesNotInFetchText.isEmpty)
               noFetchItemsAlreadyInBag(bagDir, itemsFromBagStore)
-                .flatMap(_ => validateChecksumsFetchItems(bag, itemsFromBagStore))
+                .flatMap(_ => bagStoreSettings.map(implicit bs => validateChecksumsFetchItems(bag, itemsFromBagStore))
```

Comment:
Another place where the implicit is defined with a different name than the parameter of the function that uses the implicit. @rvanheest : are there best practices for this? I find it confusing and I think it obfuscates the code.